.github/workflows/examples-java-push.yml

```
@@ -55,3 +55,24 @@ jobs:
           name: examples-java-reports
           path: '**/build/reports/*'
           retention-days: 5
+  deploy:
+    environment: sandbox
+    runs-on: ubuntu-18.04
+    needs: test
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/download-artifact@v2
+        with:
+          name: examples-java-libs
+      - name: Deploy to PaaS
+        uses: citizen-of-planet-earth/cf-cli-action@v2
+        with:
+          cf_api: https://api.london.cloud.service.gov.uk
+          cf_username: ${{ secrets.CF_USER }}
+          cf_password: ${{ secrets.CF_PASSWORD }}
+          cf_org: cabinet-office-cddo-dsa-apis
+          cf_space: sandbox
+          command: push -f examples/java/spring-boot/manifest.yml -p examples/java/spring-boot/build/libs/spring-boot-0.1-SNAPSHOT.jar
```

Can we do something like:
```suggestion
          command: push -f examples/java/spring-boot/manifest.yml -p examples/java/spring-boot/build/libs/spring-boot-*.jar
```
So we don't change this every time we bump versions?

---

We shouldn't need them, no.

One thing we can do instead is get Spring Boot's JAR name to not include version, then it's predictable?